quantus/metrics/robustness/continuity.py

```diff
@@ -98,6 +99,8 @@ def __init__(
             Indicates whether a tqdm-progress-bar is printed, default=False.
         default_plot_func: callable
             Callable that plots the metrics result.
+        return_nan_when_prediction_changes: boolean
+            When set to true, metric will be evaluated to NaN in case prediction changes after perturbation applied.
```

Comment:
Update to:
When set to true, the metric will be evaluated to NaN if the prediction changes after the perturbation is applied.

Reply:
In all places :)

Reply:
done